packages/form/src/TelephoneNumber/TelephoneNumber.js

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import InputLabel from '../InputLabel';
+import TextField from '../TextField';
+import { default as Select } from '../Select';
+import countryCodes from './TelephoneNumber.const';
+
+import './TelephoneNumber.scss';
+
+const stateClasses = {
+  success: 'has-success',
+  warning: 'has-warning',
+  error: 'has-error',
+};
+
+type InputStates = "success" | "warning" | "error";
+type InputTypes = "small" | "normal" | "large";
+
+type Props = {
+  id?: string,
```

Comment:
id should be required

Reply:
I'll make it required but note that other form elements haven't marked it as required also. It should probably be changed there as well.